ui/src/onboarding/components/CompletionStep.tsx

```diff
@@ -19,27 +32,63 @@ class CompletionStep extends PureComponent<OnboardingStepProps> {
   }
 
   public render() {
-    const {onExit, onDecrementCurrentStepIndex} = this.props
+    const {onExit} = this.props
 
     return (
       <div className="wizard--bookend-step">
+        <h3 className="wizard-step--title">You are ready to go!</h3>
+        <h5 className="wizard-step--sub-title">
+          Your InfluxDB 2.0 has 1 organization, 1 user, and 1 bucket.
```

Comment:
is this always going to be hardcoded? I think there's actually no way for them to have more than one of each of these right? If it needs to be dynamic later on, can we make an issue for it?

Reply:
They can only create 1 during onboarding so it's fine if it's hardcoded.